tests/functional/tests/management/test_disable_cleaner.py

```diff
@@ -57,7 +80,61 @@ def test_load_cleaner_disabled(pyocf_ctx):
       - disable_cleaner::load_cleaner_disabled
       - disable_cleaner::cleaning_section_alocation
     """
-    pass
+    cache_device = RamVolume(S.from_MiB(50))
+    core_device = RamVolume(S.from_MiB(10))
+
+    cache = Cache.start_on_device(cache_device, disable_cleaner=True)
+
+    core = Core.using_device(core_device)
+    cache.add_core(core)
+    vol = CoreVolume(core, open=True)
+
+    write_data = Data.from_string("Verify load")
+
+    io = vol.new_io(
+        cache.get_default_queue(), S.from_sector(3).B, write_data.size, IoDir.WRITE, 0, 0
+    )
+    io.set_data(write_data)
+
+    cmpl = OcfCompletion([("err", c_int)])
+    io.callback = cmpl.callback
+    io.submit()
+    cmpl.wait()
+    assert 0 == int(cmpl.results["err"])
```

Comment:
This test objective is to just check if disable_cleaner option is preserved between loads. No data consistency check is needed for that.

Reply:
I removed checking data consistency part with io